### PR TITLE
added linter xo. fixed linting errors

### PR DESCRIPTION
**Pull request by amok**

### CODE REVIEW

**Review by alexey-pelykh — COMMENTED** [review]
Added few comments and request is a bit outdated

File: lib/index.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
-const Source = require('./source');
```
Comment (alexey-pelykh) [inline lib/index.js:1]: Not sure if having `;` is bad

Comment (amok) [reply lib/index.js:1]: http://blog.namangoel.com/considering-the-npm-styleguide

Comment (amok) [reply lib/index.js]: http://blog.namangoel.com/considering-the-npm-styleguide

Comment (amok) [reply lib/index.js]: http://blog.namangoel.com/considering-the-npm-styleguide

**Pull request closed without merging**